packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -557,17 +561,19 @@ void main() {
       ),
     ));
 
-    final Offset floatingActionButtonOriginBottomCenter = tester.getCenter(find.byType(FloatingActionButton));
+    final Offset fabOriginBottomRight = tester.getBottomRight(find.byType(FloatingActionButton));
```

Comment:
nit: I confused `origin` and `original` here, since `origin` can also be interpreted as the reference point of some coordinate system :) Maybe we can make this clearer this way:
```dart
    // Get the Rect of the FAB to compare after the SnackBar appears.
    final Offset originalFabRect = tester.getRect(find.byType(FloatingActionButton));
    
    // Code that makes SnackBar appear.
    
    // FAB should shift upwards after SnackBar appears.
    expect(originalFabRect.getBottomRight.dy > fabBottomRight.dy, true);
```

Reply:
Fixed.